MekHQ/src/mekhq/gui/dialog/PartsStoreDialog.java

```diff
@@ -395,7 +395,7 @@ public boolean include(Entry<? extends PartsTableModel, ? extends Integer> entry
                         && !part.getName().toLowerCase().contains(txtFilter.getText().toLowerCase())
                         && !part.getDetails().toLowerCase().contains(txtFilter.getText().toLowerCase())) {
                     return false;
-                } else if ((part.getTechBase() == Part.T_CLAN)
+                } else if (((part.getTechBase() == Part.T_CLAN) || part.isClan())
```

Comment:
Wow this tech base thing is confusing. In Part there's also:

isClanTechBase() {
        return getTechBase() == TECH_BASE_CLAN;
    }

But Im not much at home there so I'm going to assume this is all as it should be.

Reply:
This is an area of the code I'm not the most at home with too, but the method is protected currently and I'd like to avoid changing most part code where possible.